src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -813,6 +814,30 @@ private IEnumerable<CompletionItem> GetArrayItemCompletions(SemanticModel model,
             return GetValueCompletionsForType(arrayType.Item.Type, context.ReplacementRange, loopsAllowed: false);
         }
 
+        private IEnumerable<CompletionItem> GetFunctionParamCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.FunctionArgument) ||
+                context.FunctionCall is not {} functionCall ||
+                model.GetSymbolInfo(functionCall) is not FunctionSymbol functionSymbol)
+            {
+                return Enumerable.Empty<CompletionItem>();
+            }
+
+            var argIndex = context.FunctionArgument is null ? 0 : functionCall.Arguments.IndexOf(context.FunctionArgument);
```

Comment:
>IndexOf [](http://example.com/codeflow?start=89&length=7)

Would it make sense to put this type computation logic in the `DeclaredTypeManager` and only retrieve here?